scania-digital-design-system/tegel
# feat(divider): prefix change

**Describe pull-request**
Changes prefix for Divider.

**Solving issue**  
Fixes: [DTS-1635](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1635)

**How to test**  
1. Go to Storybook
2. Check in Components -> Divider
3. Check the components APIs and styles.

**Suggested test steps**

[DTS-1635]: https://tegel.atlassian.net/browse/DTS-1635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
